src/ray/object_manager/common.cc

```diff
@@ -168,7 +168,7 @@ Status PlasmaObjectHeader::ReadAcquire(
     const std::unique_ptr<std::chrono::steady_clock::time_point> &timeout_point) {
   RAY_CHECK(sem.header_sem);
 
-  RAY_RETURN_NOT_OK(TryToAcquireSemaphore(sem.header_sem, timeout_point));
+  RAY_RETURN_NOT_OK(TryToAcquireSemaphore(sem.header_sem));
```

Comment:
the below busy waiting TryToAcquireSemaphore needs timeout because we want to stop busy waiting when time is out. But this part doesn't need timeout.